src/features/embed-gist-inline.js

```diff
@@ -1,30 +1,29 @@
-import { h } from 'dom-chef';
```

Comment:
Why was this removed? I think it’s required.

Reply:
Yes, good catch, my brain was still in iframe refactor mode when I deleted it